packages/core/bootstrap/src/lib/external-adapter/requester.ts

```diff
@@ -46,17 +46,18 @@ export class Requester {
       }
 
       let response: AxiosResponse<T>
+      const endpoint = `${config.baseURL}/${config.url}`
```

Comment:
Should we use `path.join()` instead to avoid weird paths if duplicate "/"s?

Reply:
resolved in #1058